docs/dapp/tools/band.md

```diff
@@ -40,11 +46,11 @@ Clicking the `getMultiPrices` button returns multiple quotes in the same call, W
 
 The `savePrice` function will save any base/quote rate that is passed to it in the storage variable named `price`. This storage data will only be updated when the “savePrice” function is called, so the saved `price` value will go stale unless this function is called repeatedly.
 
-![Save Price](../images/emerald/band_saveprice.png)
+![Save Price](../images/tools/band_saveprice.png)
 
 ### Mainnet Reference Data Contract
 
-You can access the reference data aggregator contract on Mainnet at [0xDA7a001b254CD22e46d3eAB04d937489c93174C3](https://explorer.oasis.io/mainnet/emerald/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3).
+You can access the reference data aggregator contract on Mainnet at [0xDA7a001b254CD22e46d3eAB04d937489c93174C3](https://explorer.oasis.io/mainnet/sapphire/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3).
```

Comment:
```suggestion
You can access the reference data aggregator contract on the Mainnet at [0xDA7a001b254CD22e46d3eAB04d937489c93174C3](https://explorer.oasis.io/mainnet/sapphire/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3).
```

Reply:
"Mainnet" somehow rolls of the tongue more for me than "the Mainnet." Maybe we can be more specific and just reference the Paratime with "on the Sapphire Mainnet."

```suggestion
You can access the reference data aggregator contract on the Sapphire Mainnet at [0xDA7a001b254CD22e46d3eAB04d937489c93174C3](https://explorer.oasis.io/mainnet/sapphire/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3).
```